src/main/java/falcon/io/service/dto/LiteralSplit.java

```diff
@@ -0,0 +1,19 @@
+package falcon.io.service.dto;
+
+import lombok.Builder;
+
+public class LiteralSplit extends WordSplit {
+    private String type = "LiteralSplit";
```

Comment:
This is not good because type exist also in WordSplit. It would be nicer if you had an abstract BaseSplit class that takes the type filed in the constructor and let WordSplit and LiteralSplit extend this class and call super with the correct type.